src/box/memtx_tx.c

```diff
@@ -2774,8 +2774,10 @@ memtx_tx_tuple_clarify_impl(struct txn *txn, struct space *space,
  * Detect whether the transaction can see prepared, but unconfirmed commits.
  */
 static bool
-detect_whether_prepared_ok(struct txn *txn)
+detect_whether_prepared_ok(struct txn *txn, struct space *space)
```

Comment:
Please rebase. There's one more place where this function is used now.

Reply:
Done.